assets/js/components/HostDetails/HostDetailsPage.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { networkClient } from '@lib/network';
+
+import { TARGET_HOST } from '@lib/model';
+
+import { getClusterByHost } from '@state/selectors/cluster';
+import { getInstancesOnHost } from '@state/selectors/sapSystem';
+
+import { getHost, getHostSelectedChecks } from '@state/selectors/host';
+import { isSaving } from '@state/selectors/checksSelection';
+import { hostExecutionRequested } from '@state/actions/lastExecutions';
+
+import { deregisterHost } from '@state/hosts';
+import HostDetails from './HostDetails';
+
+// eslint-disable-next-line no-undef
+const { grafanaPublicUrl } = config;
+
+function HostDetailsPage() {
+  const { hostID } = useParams();
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const host = useSelector(getHost(hostID));
+  const cluster = useSelector((state) => getClusterByHost(state, hostID));
+  const sapInstances = useSelector((state) =>
+    getInstancesOnHost(state, hostID)
+  );
+
+  const hostSelectedChecks = useSelector((state) =>
+    getHostSelectedChecks(state, hostID)
+  );
+  const saving = useSelector(isSaving(TARGET_HOST, hostID));
+
+  const [exportersStatus, setExportersStatus] = useState([]);
+
+  const getExportersStatus = async () => {
+    const { data } = await networkClient.get(
+      `/hosts/${hostID}/exporters_status`
+    );
+    setExportersStatus(data);
+  };
+
+  useEffect(() => {
+    getExportersStatus();
+  }, []);
+
+  if (!host) {
+    return <div>Not Found</div>;
+  }
+
+  return (
+    <HostDetails
+      agentVersion={host.agent_version}
+      cluster={cluster}
+      deregisterable={host.deregisterable}
+      deregistering={host.deregistering}
+      exportersStatus={exportersStatus}
+      grafanaPublicUrl={grafanaPublicUrl}
+      heartbeat={host.heartbeat}
+      hostID={host.id}
+      hostname={host.hostname}
+      provider={host.provider}
+      providerData={host.provider_data}
+      sapInstances={sapInstances}
+      savingChecks={saving}
+      selectedChecks={hostSelectedChecks}
+      slesSubscriptions={host.sles_subscriptions}
+      cleanUpHost={() => {
+        dispatch(
+          deregisterHost({ id: hostID, hostname: host.hostname, navigate })
+        );
+      }}
+      requestHostChecksExecution={() => {
+        dispatch(hostExecutionRequested(host, hostSelectedChecks, navigate));
```

Comment:
Some doubts on this. If we should get the `host` and `hostSelectedChecks` as function arguments, instead of "global" variables from the file.
This would force me to pass the whole `host` object to the details view as this is needed by the state function